MAUI/Picker/populating-items.md

```diff
@@ -126,38 +126,38 @@ Selected item text style can be customized by setting `SfPicker.SelectedTextStyl
 
     this.Picker.SelectedTextStyle.FontSize = 16;
     this.Picker.SelectedTextStyle.FontAttributes = FontAttributes.Bold;
-    this.Picker.SelectedTextStyle.TextColor = Colors.Gray;
+    this.Picker.SelectedTextStyle.TextColor = Colors.White;
 
 {% endhighlight %}
 {% endtabs %}
 
 ### Unselected item customization
 
-Unselected item text style can be customized by setting `SfPicker.TextStyle` property.
+Unselected item [TextStyle](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Picker.PickerBase.html#Syncfusion_Maui_Picker_PickerBase_TextStyle) can be customized by setting `SfPicker.TextStyle` property.
 
 {% tabs %}
 {% highlight xaml tabtitle="MainPage.xaml" %}
 
     <sfPicker:SfPicker x:Name="Picker">
         <sfPicker:SfPicker.TextStyle>
-            <sfPicker:PickerTextStyle FontSize="17" FontAttributes="Italic" TextColor="Green">
+            <sfPicker:PickerTextStyle FontSize="16" FontAttributes="Italic" TextColor="Black">
             </sfPicker:PickerTextStyle>
         </sfPicker:SfPicker.TextStyle>
     </sfPicker:SfPicker>
 
 {% endhighlight %}
 {% highlight c# tabtitle="MainPage.xaml.cs" %}
 
-    this.Picker.TextStyle.FontSize = 17;
+    this.Picker.TextStyle.FontSize = 16;
     this.Picker.TextStyle.FontAttributes = FontAttributes.Italic;
-    this.Picker.TextStyle.TextColor = Colors.green;
+    this.Picker.TextStyle.TextColor = Colors.Black;
 
 {% endhighlight %}
 {% endtabs %}
 
 ## Adding custom view to items
 
-In picker control, the items can be customized with a custom view of all items by ItemTemplate and assign a custom view to the `ItemTemplate`.
+In picker control, the items can be customized with a custom view of all items by [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Picker.SfPicker.html#Syncfusion_Maui_Picker_SfPicker_ItemTemplate) and assign a custom view to the `ItemTemplate`.
```

Comment:
In picker control, the items can be customized with a custom view of all items by the [ItemTemplate](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Picker.SfPicker.html#Syncfusion_Maui_Picker_SfPicker_ItemTemplate) and assign a custom view to the `ItemTemplate`.

Reply:
Content modified.